docs/.vuepress/config.js

```diff
@@ -80,17 +80,16 @@ module.exports = {
           ],
         },
         {
-          title: "Quick Start",
+          title: "Install",
           collapsable: false,
-          path: "/docs/quick-start/",
+          path: "/docs/install/",
```

Comment:
I'd like to add redirects for the old urls to `_redirects` in `docs/.vuepress/public/_redirects`

Reply:
Good catch, added in https://github.com/pomerium/pomerium/pull/2406/commits/1a5aa5187b42110e46a5eba0edb2d9ace51da295